{{cookiecutter.package_name}}/README.rst

```diff
@@ -23,7 +23,7 @@ Once you have all three installed, you should open up your normal shell
 You'll then make an environment, clone this repository, then install
 all necessary requirements as follows::
 
-  :~$ conda create --name={{ cookiecutter.package_name }} python=3.6
+  :~$ conda create --name=vivarium_ciff_sam python=3.8
```

Comment:
Don't we still want the template here?  Good on the update to python 3.8.

Reply:
oops yes